liquidsoap/config/main.liq

```diff
@@ -34,6 +34,18 @@ def write_metadata_to_file(m) =
     system("echo "^ quote(m["artist"] ^" - "^ m["title"]) ^" >> /data/history.txt")
 end
 
+def check_emission_time() =
+    localtime(time(), fun (~sec,~min,~hour,~mday,~mon,~year,~wday,~yday,~isdst) ->
+    mhour == 19 and min == 30) 
+end
+
+def emission_polling(source) =
+    if check_emission_time then
+        request.queue(id=source)
+    end
+    false
+end
```

Comment:
You need to pass a function of time: `()->float` to `add_timeout` so something like this:
```
def emission_polling(id) =
  def check() =
    if 19h30m then
        server.execute("#{id}.push <path/to/emission.mp3>")
    end
    # Retry every 10 seconds
    10.
  end
  check
end
```

Reply:
Hmm this needs to be smarter otherwise you will push the same request multiple times during your window. You should check every `40.` seconds I believe.